apim/3.x/README.adoc

```diff
@@ -37,42 +37,62 @@ $ helm package .
 
 === Installing the Chart
 
-To install the chart from the Helm repository with the release name
-`+graviteeio-apim3x+`:
+
+Now, install the chart from the Helm repo with the release name
+
+`+graviteeio-apim3x+`, run:
 
 [source,bash]
 ----
 $ helm install --name graviteeio-apim3x graviteeio/apim3
```

Comment:
This is a helm v2 command, the helm v3 is without the --name parameter : 
`helm install graviteeio-apim3x graviteeio/apim3`